### ISSUE TITLE
Using "Static Lightmaps" light baking mode on imported 3D scene breaks normal mapping direction

### ISSUE THREAD

**Issue by wareya** [godotengine/godot#83132]
### Godot version

4.1.1 stable

### System information

Windows 10, amd rx 480 gpu

### Issue description

When I turn on the "Static Lightmaps" light baking mode on an imported model where some faces have inverted UV coordinates, the tangents (or binormals) get inverted, so normal mapping comes out wrong:

![Godot_v4 1 1-stable_win64_2023-10-10_21-44-19](https://github.com/godotengine/godot/assets/585488/4d6ea886-5789-4e8f-991b-808d39cd84b0)
![Godot_v4 1 1-stable_win64_2023-10-10_21-44-23](https://github.com/godotengine/godot/assets/585488/6caab9ab-20a3-4a0e-8935-c069f99dace6)

### Steps to reproduce

Import a model with normal mapping where some faces have inverted UVs. Enable Static Lightmaps mode and reimport. Watch the normal mapping direction change.

### Minimal reproduction project

Test model (gltf, not a whole project):
[testsphere.zip](https://github.com/godotengine/godot/files/12863973/testsphere.zip)

**Comment by Calinou** [comment]
Can you reproduce this with **Generate Lods** disabled in the Import dock?

**Comment by wareya** [comment]
Yes. None of the three settings above Light Baking seem to have any effect on the normal mapping direction.

**Comment by BlueCube3310** [comment]
Does this still happen after #88725?

**Comment by Calinou** [comment]
I can't reproduce this on `master` (light is coming from the left):

![Image](https://github.com/user-attachments/assets/c450d7a5-0fa2-4eba-b96d-9b1199eb96f6)
